templates/pkg/crd_sdk.go.tpl

```diff
@@ -94,6 +94,7 @@ func (rm *resourceManager) sdkFind(
 		return nil, err
 	}
 {{ $setCode := GoCodeSetReadManyOutput .CRD "resp" "ko" 1 }}
+{{ $setCustomOutputOperation := .CRD.GetCustomSetOutputOperation .CRD.Ops.ReadMany }}
```

Comment:
any reason to place this here instead of below where it is used?

Reply:
Moved the code and used `{{ if $setOutputCustomMethodName := ` to combine into one statement.